test/env.js

```diff
@@ -1,9 +1,10 @@
 process.env.METAMASK_ENV = 'test';
 
 /**
- * Used for testing components that use the Icon component 
+ * Used for testing components that use the Icon component
  * 'ui/components/component-library/icon/icon.js'
  */
 process.env.ICON_NAMES = {
   LOADING_FILLED: 'loading-filled',
+  CLOSE_OUTLINE: 'close-outline',
```

Comment:
This still does not fix the propType issue as ICON_NAMES are still returning as undefined. Need to find a solution 
issue https://github.com/MetaMask/metamask-extension/issues/16046
PR https://github.com/MetaMask/metamask-extension/pull/16078